src/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import {
+  SpinnerDefaultProps,
+  SpinnerProps,
+  SpinnerState,
+  SPINNER_TYPES,
+} from './types';
+import { st, classes } from './Spinner.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+
+export class Spinner extends React.Component<SpinnerProps, SpinnerState> {
+  static displayName = 'Spinner';
+  static defaultProps: SpinnerDefaultProps = {
+    type: SPINNER_TYPES.regular,
+    width: 100,
```

Comment:
By looking in the zeroheight spec, I think the default width should be `50px`
Also, maybe instead of `width` we can call it `diameter`? since it's only one prop

Reply:
Fixed